packages/oas/src/index.ts

```diff
@@ -724,7 +724,7 @@ export default class Oas {
     Object.keys(api.webhooks ? api.webhooks : []).forEach(id => {
       webhooks[id] = {} as Record<RMOAS.HttpMethods, Webhook>;
       Object.keys(api.webhooks[id]).forEach((method: RMOAS.HttpMethods) => {
-        webhooks[id][method] = this.operation(id, method, { isWebhook: true });
+        webhooks[id][method] = this.operation(id, method, { isWebhook: true }) as Webhook;
```

Comment:
TypeScript was upset that we were assigning an `Operation` class to a `Webhook` class so added this since it should always be a `Webhook` and never an `Operation` in this function.